edge-modules/TwinTester/TwinEdgeOperationsResultHandler.cs

```diff
@@ -43,27 +47,23 @@ public Task HandleTwinValidationStatusAsync(string status)
 
         public Task HandleReportedPropertyUpdateExceptionAsync(string propertyKey, string failureStatus)
         {
-            return this.SendReportAsync(StatusCode.ReportedPropertyUpdateCallFailure, propertyKey, failureStatus);
+            TwinCollection properties = this.GetTwinCollection(propertyKey);
```

Comment:
Why do we need to report fail twin update? is this used for long haul test?

Reply:
It was for long haul and I also reported for connectivity, but since it is not used in connectivity I removed it. This class handles connectivity reporting so it's fine to remove it.